server/manager/src/test/java/org/apache/accumulo/manager/upgrade/Upgrader11to12Test.java

```diff
@@ -599,17 +599,25 @@ public void testMoveTableProperties() throws Exception {
     sysProps.put(Property.TABLE_BLOOM_ENABLED.getKey(), "true");
     sysProps.put(Property.TABLE_BLOCKCACHE_ENABLED.getKey(), "true");
     sysProps.put(Property.TABLE_CLASSLOADER_CONTEXT.getKey(), "sysContext");
+    sysProps.put(Property.TABLE_ITERATOR_PREFIX.getKey() + "test", "iteratorProperty");
```

Comment:
Could also test a constraint property.  Ran this test w/ coverage and noticed it was not hitting that case.

Reply:
Added in 7ddd50d